app/routes/dog.js

```diff
@@ -11,6 +11,10 @@ module.exports = [{
     const indexNumber = request.params.indexNumber
     try {
       const dog = await getDogByIndexNumber(indexNumber)
+
+      if (dog === null) {
+        return h.response().code(404)
+      }
       return h.response({ dog: dogDto(dog) }).code(200)
```

Comment:
Very picky but a blank line is nice before the 'return'

Reply:
Good call , have done the same to you